src/dune_rpc_client/where.ml

```diff
@@ -47,10 +47,7 @@ let to_socket = function
   | `Ip (`Host host, `Port port) -> Unix.ADDR_INET (Unix.inet_addr_of_string host, port)
 ;;
 
-let to_string = function
-  | `Unix p -> sprintf "unix://%s" p
-  | `Ip (`Host host, `Port port) -> sprintf "%s:%d" host port
-;;
+let to_string t = Dune_rpc_private.Where.to_string t
```

Comment:
Use the right "dbus" format so that it can be parsed back in https://github.com/ocaml/dune/blob/071cb4718d4c23f9b3645c8c254028df6d1a35c2/otherlibs/dune-rpc/private/where.ml#L154-L162